frontend/src/app/modules/external-booking/external-booking-choose-time-page/external-booking-choose-time-page.component.ts

```diff
@@ -19,7 +21,7 @@ import { TZone } from 'moment-timezone-picker';
     templateUrl: './external-booking-choose-time-page.component.html',
     styleUrls: ['./external-booking-choose-time-page.component.sass'],
 })
-export class ExternalBookingTimeComponent extends BaseComponent implements OnInit {
+export class ExternalBookingTimeComponent extends BaseComponent implements OnInit, OnDestroy {
     link: string;
 
```

Comment:
Why do you use on destroy, BaseComponent already implements it&

Reply:
Fixed